### ISSUE TITLE
issue with cow

### ISSUE THREAD

**Issue by samoht** [dsheets/codoc#45]
```
#=== ERROR while installing codoc.0.2.0 =======================================#
# opam-version 1.2.1
# os           darwin
# command      make
# path         /Users/thomas/.opam/doc/build/codoc.0.2.0
# compiler     4.02.1+doc
# exit-code    2
# env-file     /Users/thomas/Downloads/codoc-64891-d2d111.env
# stdout-file  /Users/thomas/Downloads/codoc-64891-d2d111.out
# stderr-file  /Users/thomas/Downloads/codoc-64891-d2d111.err
### stdout ###
# codocEnvironment.cmx      <= compile-native unit-codocEnvironment-lib-codoc
# [...]
# codoc.cmxa codoc.a        <= archive-native lib-codoc
# codoc.cmxs                <= archive-shared lib-codoc
# codocCli.cmi              <= compile-byte unit-codocCli-bin-codoc
# codocConfig.cmi codocConfig.cmo <= compile-byte unit-codocConfig
# codocCli.cmo              <= compile-byte unit-codocCli-bin-codoc
# codocSysUtil.cmi codocSysUtil.cmo <= compile-byte unit-codocSysUtil
# codocCliExtract.cmi codocCliExtract.cmo <= compile-byte unit-codocCliExtract-bin-codoc
# codocCliLink.cmi codocCliLink.cmo <= compile-byte unit-codocCliLink-bin-codoc
# codocCliHtml.cmi codocCliHtml.cmo <= compile-byte unit-codocCliHtml-bin-codoc
### stderr ###
# File "_build/bin-codoc/codocCliHtml.ml", line 69, characters 2-21:
# Error: Unbound value Cow.Html.output_doc
# make[1]: *** [_build/bin-codoc/codocCliHtml.cmo] Error 2
# make: *** [all] Error 2
```

**Comment by avsm** [comment]
you need cow master branch (we should release that @dsheets, its working well for me)

**Comment by dsheets** [comment]
On it.

**Comment by dsheets** [comment]
Resolved by ocaml/opam-repository#3531.
